api/src/main/java/org/openmrs/module/aijar/metadata/core/EncounterTypes.java

```diff
@@ -87,19 +87,38 @@ public String uuid() {
         }
     };
 
+    
+     public static EncounterTypeDescriptor CHILD_ENCOUNTER_TYPE = new EncounterTypeDescriptor() {
+        @Override
+        public String name() {
```

Comment:
Change name - Visit of a child to a health facility for immunization

Reply:
Noted. To be updated.

Reply:
@ssmusoke, shouldn't "Visit of a child for immunization and other services" be the description and name be "Child Immunization Encounter"?

Reply:
@faithNassiwa That is correct